src/storage/store/account/link_store.rs

```diff
@@ -205,12 +182,12 @@ impl LinkStore {
                 }
 
                 Ok(false)
-            })?;
+            },
+        )?;
 
-        let messages_bytes =
-            message::get_many_messages_as_bytes(store.db().borrow(), message_keys)?;
+        let messages_bytes = get_many_messages_as_bytes(store.db().borrow(), message_keys)?;
         let next_page_token = if last_key.len() > 0 {
-            Some(last_key[prefix.len()..].to_vec())
```

Comment:
Hmm, why did this change?

Reply:
This changed at the db level when I pulled the db code in. The page token is the whole key and not a piece of it. Only providing the suffix in the token worked when we only iterated over a single prefix (e.g. fid) but now that we have a start and stop we can't keep only some suffix.